### PR TITLE
Added LifeBeyondTheBorder episode - 02.

### PR DESCRIPTION
**Pull request by Abdus-Samee**

### CODE REVIEW

File: _activities/Lifebeyondtheborders/overview.md [hunk 1]
```diff
@@ -1,10 +1,7 @@
...
 layout: sessionTemplate
 title: Overview 
-desc: >
- Here goes description + written question ans  lkasjfdlk l asdklfljslkad fklsf jkllasdjfk safjdas fjsklf jkldfjlk
- klsjfd lajf kldsjflkas jfklasdjflk sdjflk jflksjf skdfjsd lakfjaskl dflksajflkas flkdasjflkdsjflksadjf
- lkfjaskldfjlksajf lksajfdkl fjaslkfjaklfjlksda flkasjf
- sdlkfjalksdfj lkasjfdlksdjdflkasjdflkadsjf
- slakjdflksdjkflkj
+desc: 
+    This section covers the interview of alumni from foreign countries focusing on their life and job. The interview is provided in the format of
```
Comment (imranhasanhira) [inline _activities/Lifebeyondtheborders/overview.md]: Good job in adding some basic overview 👍

File: _activities/Lifebeyondtheborders/002.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
+---
```
Comment (imranhasanhira) [inline _activities/Lifebeyondtheborders/002.md]: Can we change the folder name from camelcase to dash-separated ?

Comment (Abdus-Samee) [reply _activities/Lifebeyondtheborders/002.md]: I've changed the styling of the folder name...

Comment (imranhasanhira) [reply _activities/Lifebeyondtheborders/002.md]: it still looks like camel case. we need the url to be like dash-sparated-lower-cased

Comment (Abdus-Samee) [reply _activities/Lifebeyondtheborders/002.md]: I've opened a new pull request. Updated the links in that.

**Pull request closed without merging**